util/dataframe_ops.py

```diff
@@ -38,7 +38,7 @@ def compute_rel_diff_dataframe(df1, df2):
     return out
 
 
-def compute_div_dataframe(df1, df2):
+def compute_division_df(df1, df2):
```

Comment:
I would prefer to then call it just `compute_division`. To make it explicit that it is supposed for dataframes, one could also add type annotations.